buildchain/buildchain/salt_tree.py

```diff
@@ -221,8 +221,10 @@ def _get_parts(self) -> Iterator[str]:
     Path('salt/metalk8s/addons/ui/deployed/namespace.sls'),
     Path('salt/metalk8s/addons/ui/deployed/ui.sls'),
 
-    Path('salt/metalk8s/addons/solutions/deployed.sls'),
-    Path('salt/metalk8s/addons/solutions/environment-crd.sls'),
+    Path('salt/metalk8s/addons/solutions/deployed/init.sls'),
+    Path('salt/metalk8s/addons/solutions/deployed/environment-crd.sls'),
```

Comment:
Oh, an *environment* is an actual CRD? I though it maps to a *Namespace*?

Reply:
No, an `Environment` is actually a "group" of `Namespace`s with information about which version of a Solution (Operator) is running in such `Namespace`s (one namespace per Solution, to keep things organized by default). So it's a CRD (for simplicity, although we may want to change this in the future), but it's not tied to any automatic action through a controller.

Reply:
This concept was introduced to help users unfamiliar with namespaces to correctly manage Solutions without using a single namespace for a bunch of Solutions (the UI enforces a single Operator per Solution in each Environment, so actually each Operator has its own Namespace)